### PR TITLE
Reduce package sizes + move configs to a more canonical location

### PR DESCRIPTION
**Pull request by basilisk487**

### CODE REVIEW

File: pkg/after-install.sh [hunk 1]
```diff
@@ -36,6 +39,8 @@ if [[ -f $old_pid_file ]]; then
 	rm $old_pid_file
 fi
 
+curl -s https://github.com/wavefrontHQ/java/raw/master/pkg/install_jre.sh | bash
```
Comment (moribellamy) [inline pkg/after-install.sh]: use a release branch?

File: pkg/build.sh [hunk 2]
```diff
@@ -38,10 +38,12 @@ fpm \
 	--before-remove before-remove.sh \
 	--after-remove after-remove.sh \
 	--architecture amd64 \
-	--config-files opt/wavefront/wavefront-proxy/conf \
+	--config-files etc/wavefront/wavefront-proxy \
```
Comment (moribellamy) [inline pkg/build.sh]: everything in this directory will be a sticky conf file -- is that what you want? (i think earlier we talked about keeping a .default config file somewhere)

File: pkg/etc/init.d/wavefront-proxy [hunk 3]
```diff
@@ -25,8 +25,21 @@ desc=${DESC:-Wavefront Proxy}
...
 proxy_dir=${PROXY_DIR:-$wavefront_dir/wavefront-proxy}
+config_dir=${CONFIG_DIR:-/etc/wavefront/wavefront-proxy}
 export JAVA_HOME="$proxy_dir/jre"
-conf_file=${CONF_FILE:-$proxy_dir/conf/wavefront.conf}
+conf_file=$CONF_FILE
+if [[ -z $conf_file ]]; then
+    legacy_config_dir=$proxy_dir/conf
+    if [[ -r "$legacy_config_dir/wavefront.conf" ]]; then
+        conf_file="$legacy_config_dir/wavefront.conf"
+		>&2 echo "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
```
Comment (moribellamy) [inline pkg/etc/init.d/wavefront-proxy]: extra indent?

File: pkg/stage.sh [hunk 4]
```diff
@@ -45,9 +45,6 @@ cp -r opt build/opt
 cp -r etc build/etc
 cp -r usr build/usr
 
-echo "Stage the JRE..."
```
Comment (moribellamy) [inline pkg/stage.sh]: update README.md to reflect that you no longer CP the JRE into the docker container

Comment (basilisk487) [inline pkg/stage.sh]: README.md updates as per our conversation

File: pkg/etc/init.d/wavefront-proxy (hunk 3 above)
Comment (basilisk487) [inline pkg/etc/init.d/wavefront-proxy]: Fixed tabs vs spaces

File: pkg/build.sh (hunk 2 above)
Comment (basilisk487) [inline pkg/build.sh]: Fixed - .default configs are used

File: pkg/after-install.sh (hunk 1 above)
Comment (basilisk487) [inline pkg/after-install.sh]: Done!

**Comment by basilisk487** [comment]
@mori: updated as per our conversation

File: pkg/README.md [hunk 5]
```diff
@@ -17,16 +17,22 @@ Build and run the docker container for building.
...
-    docker cp my_container:/root/java/pkg/wavefront-proxy_3.8-1_amd64.deb .
+    docker cp my_container:/root/java/pkg/wavefront-proxy_3.99-4_amd64.deb .
 
-This will build the agent from head and package it into a deb or an rpm. The agent will use whatever JRE
-you bundle with it -- Wavefront uses Zulu 8.13.0.5 0f21d10ca4f1.
+This will build the agent from head and package it into a deb or an rpm. The after-install.sh script will download
+and install Zulu 8.13.0.5 0f21d10ca4f1 JRE by default, so to bundle the package with a JRE of your choice instead,
+run the following commands inside the docker container right before build.sh:
+
+```
```
Comment (moribellamy) [inline pkg/README.md]: if this is equivalent to indenting, please indent to maintain consistency

Comment (basilisk487) [inline pkg/README.md]: Fixed!

**Pull request merged**